src/main/java/org/spongepowered/common/mixin/invalid/core/util/datafix/DataFixerMixin.java

```diff
@@ -31,6 +31,7 @@
 import org.spongepowered.asm.mixin.injection.Constant;
 import org.spongepowered.asm.mixin.injection.ModifyConstant;
 
+// i509VCB: DataFixerUpper makes this redundant due to the entire infrastructure changing.
```

Comment:
Try and focus each commit of your PR on a theme. Such as playing with Villagers. Else things get scattered and become odd.

Also I'll be invalidating the lack of targets in the core plugin so you can exclude this.